nautobot/ui/src/views/generic/ObjectRetrieve.js

```diff
@@ -70,7 +70,7 @@ export default function ObjectRetrieve({ api_url }) {
     };
 
     return (
-        <GenericView objectData={data}>
+        <GenericView objectData={data} key={window.location.pathname}>
```

Comment:
This key value is added so that when the user clicks on the link of the changelog/note entry from `ObjectRetrieveView`. The component is re-rendered and the very first tab will become active. Without it, the URL will redirect but the page won't re-render, we will still remain on the Changelog tab/Note tab which is not ideal.